docs/user/openstack/external_load_balancer.md

```diff
@@ -0,0 +1,224 @@
+# Deploying OpenShift with an external load balancer
+
+This document explains how to deploy OpenShift with an external load balancer, rather than with the internal (that is based on HAproxy & Keepalived).
+
+## Table of Contents
+- [Deploying OpenShift with an external load balancer](#deploying-openshift-with-an-external-load-balancer)
+  - [Table of Contents](#table-of-contents)
+  - [Common prerequisites](#common-prerequisites)
+  - [Deploy the load-balancer](#deploy-the-load-balancer)
+  - [Deploy OpenShift](#deploy-openshift)
+  - [Known limitations](#known-limitations)
+  - [Notes](#notes)
+
+## Common prerequisites
+
+* When deploying OpenShift with an external load balancer, it's required to bring your own network(s) before
```

Comment:
I don't think that's technically correct. As long as the LB has a route to get to the subnets where the nodes are deployed, it should be fine, right? In my understanding these subnets don't have to exist beforehand and we can let the IPI installer create the machineSubnet.

Reply:
This is questionable. With terraform-created networks, we don't have the control on the allocation pool for example, so this makes the LB config a bit more difficult I think.
I think we should recommend BYON for simplicity. Also because that's what we have been testing so far.